dlt/destinations/impl/bigquery/factory.py

```diff
@@ -42,6 +42,7 @@ def _raw_capabilities(self) -> DestinationCapabilitiesContext:
         caps.supports_ddl_transactions = False
         caps.supports_clone_table = True
         caps.schema_supports_numeric_precision = False  # no precision information in BigQuery
+        caps.supported_merge_strategies = ["delete-insert", "scd2"]
```

Comment:
OK I think you intend to merge some kind of simple version and then add support one by one. Good!

Reply:
Correct, the plan was `snowflake` first. We get `postgres` "for free" because that's what I was using to develop the logic. Some of the other destinations might already work, but have not been tested. They will be added in a followup PR.